com.konstantintutsch.Lock.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.konstantintutsch.Lock
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: com.konstantintutsch.Lock
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=xdg-run/gnupg:ro
```

Comment:
you need this for the gpg socket? This is made available by `--socket=gpg-agent` I don't think you need both.

Reply:
You're right, `--filesystem=xdg-run/gnupg:ro` is not necessary.

e13b169ccbc5dd47f319b6482d68971c65d4e64c